client/python/gradio_client/client.py

```diff
@@ -1026,18 +1023,19 @@ def _download_file(
         if isinstance(x, str):
             file_name = utils.decode_base64_to_file(x, dir=save_dir).name
         elif isinstance(x, dict):
-            if x.get("is_file"):
-                filepath = x.get("name")
-                assert filepath is not None, f"The 'name' field is missing in {x}"
+            data = x.get("data", None)
```

Comment:
This function is only called on `V4` spaces so it's ok to delete the data check since data will no longer be present in V4 after this is merged.

Don't worry if client tests fail as long as they're connecting to a space on spaces. It's cause they're running an older version of the client.